src/net.cpp

```diff
@@ -1625,7 +1626,7 @@ void CConnman::ThreadOpenConnections()
         {
             LOCK(cs_vNodes);
             BOOST_FOREACH(CNode* pnode, vNodes) {
-                if (!pnode->fInbound) {
+                if (!pnode->fInbound && !pnode->fAddnode) {
```

Comment:
Do you think I should add a comment like, "Inbound and addnode connections don't use our limited pool of automatic outbound connections, so we don't consider them for netgroup exclusion."?